sematic/api/endpoints/meta.py

```diff
@@ -51,3 +57,59 @@ def env_endpoint(user: Optional[User]) -> flask.Response:
             continue
 
     return flask.jsonify(dict(env=env))
+
+
+@sematic_api.route("/api/v1/meta/health", methods=["GET"])
+def get_server_health() -> flask.Response:
+    """Get information about the server health.
+
+    Returns
+    -------
+    A flask response with a json payload. The payload has two fields:
+        api: status info about basic access to the API
+        db: status info about access to the DB
+    Status info for each of these fields is a dict of the form:
+    {
+        "healthy": <bool>,
+        "message": <str message about status>,
+    }
+    If any of the "healthy" fields is `False`, will return a 503 status code.
+    """
+    # healthy by nature of the fact that this API is working.
+    api_status = dict(
+        healthy=True,
+        message="API is healthy",
+    )
+
+    db_healthy = True
+    db_message = "Basic database access verified"
+    with db().get_session() as session:
+        try:
+            statement = sqlalchemy.text("SELECT id FROM runs LIMIT 1;")
+            if len(list(session.execute(statement))) > 0:
+                db_message = "Verified at least one run is present in the DB."
+
+        except Exception as e:
+            logger.fatal(
+                "Sematic is unable to access the database and "
+                "therefore will be unable to operate properly: %s",
+                e,
+            )
+            db_healthy = False
+            db_message = f"Problems accessing the database. Error: {e}"
+
+    db_status = dict(
+        healthy=db_healthy,
+        message=db_message,
+    )
+    payload = dict(
+        api=api_status,
+        db=db_status,
+    )
+
+    status = HTTPStatus.OK if db_healthy else HTTPStatus.SERVICE_UNAVAILABLE
+    logger.info("Health: %s", payload)
```

Comment:
i would make this `logger.debug`.  if there are many front-end sessions active, they'll all be polling this endpoint frequently and will end up spamming the logs unnecessarily.